Makefile

```diff
@@ -55,6 +55,10 @@ build-image-kubernetes:                   ## Build the kubernetes image
 build-image-runtime:                      ## Build the runtime image
 	./scripts/build-image-runtime
 
+.PHONY: build-s390x
+build-s390x:                                   ## Build s390x
+	./scripts/build-s390x
+
```

Comment:
I think this can go away now that the script has been removed

Reply:
done in https://github.com/rancher/rke2/pull/2171/commits/133611b8d82876240448ae1357cdf8a86f60fe60